Sources/Shout/SSH.swift

```diff
@@ -110,6 +110,11 @@ public class SSH {
         }
     }
     
+    public func sendFile(localURL: URL, remotePath: String, mode: Int) throws -> Int32 {
```

Comment:
Are there any advantages/disadvantages to implementing this func with SCP rather than SFTP as implemented by #15? I'm not too familiar with the differences between these two

Reply:
I'm not too familiar either, but [this explanation](https://superuser.com/a/134942) seems to align with the libSSH2 API. It happened to better fit my use case (putting a file on another computer) than the more powerful SFTP functionality.

If this PR was merged and both SCP and SFTP in Shout only implemented "upload" then it might seem kind of silly, but I think both implementations can coexist in Shout. If there was full coverage of libSSH2's functionality later then the difference might be more apparent to users.

Reply:
Perhaps it makes sense to extract `FilePermissions` struct from the `SFTP.swift` to a separate file so you can use it nicely. Could you do that please?

Reply:
> Perhaps it makes sense to extract FilePermissions struct from the SFTP.swift to a separate file so you can use it nicely. Could you do that please?

Yes, I plan to do that as part of this PR before it's merged 😄 I noted this in the PR description. Thanks for adding that, it's a nicer API than raw numbers!